src/dynapi/SDL_dynapi_overrides.h

```diff
@@ -297,11 +297,13 @@
 #define SDL_IntersectRectAndLine SDL_IntersectRectAndLine_REAL
 #define SDL_GetNumRenderDrivers SDL_GetNumRenderDrivers_REAL
 #define SDL_GetRenderDriverInfo SDL_GetRenderDriverInfo_REAL
+#define SDL_GetRenderDriverInfoEx SDL_GetRenderDriverInfoEx_REAL
```

Comment:
Please revert your changes to dynapi, and run `gendynapi.pl` under src/dynapi/ in order to properly generate entries for new additions.

Reply:
Sorry about that. It should be fixed now.